packages/core/src/components/pane/pane.tsx

```diff
@@ -224,11 +219,40 @@ export class Pane {
       this.parentHeightPx = entries[0].borderBoxSize[0].blockSize;
     });
     if (parentElement) this.resizeObserver.observe(parentElement);
+
+    if (this.hideOnCollapse) this.disposeListener = this.addClickOutsideEvent();
```

Comment:
Just a question: should this be only for floating panes?
Also when a hideOnCollapse pane gets closed it simply disappears without animation. Is this intended?

Reply:
hideOnCollapse can be true for inline and floating panes. It just defines if a certain pane has a collapsed state or not. The disappearance without animation is intentional.